authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc

```diff
@@ -0,0 +1,90 @@
+[id="about-cloud-credential-operator"]
+= About the Cloud Credential Operator
+include::modules/common-attributes.adoc[]
+:context: about-cloud-credential-operator
+:FeatureName: Support for Amazon Web Services Secure Token Service (AWS STS)
+toc::[]
+
+//Links that point to the entry in the RH Operators ref should now point here
+
+The Cloud Credential Operator (CCO) manages cloud provider credentials as Kubernetes custom resource definitions (CRDs). The CCO syncs on `CredentialsRequest` custom resources (CRs) to allow {product-title} components to request cloud provider credentials with the specific permissions that are required for the cluster to run.
+
+By setting different values for the `credentialsMode` parameter in the `install-config.yaml` file, the CCO can be configured to operate in several different modes. If no mode is specified, or the `credentialsMode` parameter is set to an empty string (`""`), the CCO operates in its default mode.
```

Comment:
s/in its default mode/in Mint Mode, by default ??

Reply:
sort of - except mint isn't supported on all platforms